python/cudf/cudf/core/column/datetime.py

```diff
@@ -199,7 +199,7 @@ def to_pandas(
 
         # Pandas supports only `datetime64[ns]`, hence the cast.
         return pd.Series(
-            self.astype("datetime64[ns]").to_array("NAT"),
+            self.astype("datetime64[ns]").fillna("NAT").values_host,
```

Comment:
Prefer the capitalizations `NaN` and `NaT` for consistency with their standard pandas representations.
```suggestion
            self.astype("datetime64[ns]").fillna("NaT").values_host,
```